factory-containers/publish-compose-apps

```diff
@@ -135,22 +139,55 @@ def publish(factory: str, tag: str, app_name: str) -> str:
     return app + '@sha256:' + sha.decode()
 
 
-def main(factory: str, tag: str):
+def main(factory: str, tag: str, platforms: str, sha: str, app_preload_flag: str, app_dir=None):
```

Comment:
trying to understand the addition of `app_dir`. does this make it easier for you to do unit/manual testing?

Reply:
It makes it easier to do manual testing. I can remove it.

Reply:
no. i just wanted to make sure i was reading things correctly. it seems useful.